PlayCover/en.lproj/Localizable.strings

```diff
@@ -295,3 +295,7 @@
 "alert.quota.limit" = "Quota limit reached. It may take up 24 hours to download this file again";
 "alert.notSpace" = "Not enough space on disk";
 "alert.corrupted" = "File might be corrupted. Try to download it again";
+"folder.textfield.name" = "Folder Name...";
```

Comment:
`folder` should be the localization string prefix for all things related to folder.